packages/web-runtime/src/index.ts

```diff
@@ -212,6 +212,13 @@ export const bootstrapApp = async (configurationPath: string): Promise<void> =>
         })
       }
 
+      // load sharing roles from graph API
```

Comment:
I'd also be fine keeping the previous order and removing the early return. 
```js
if (personalSpace) {
  spacesStore.updateSpaceField... for renaming the space locally to `Personal`
}
```
Is legit, I don't see the benefit of that being an early return.

Reply:
Agreed! While early returns are nice in certain situations, it can quite easily lead to such issues.